### DISCUSSION TITLE
QGEP in the field

### DISCUSSION THREAD

**Discussion by TriformMoritz** [QGEP/QGEP#731] [Q&A] [answered]
I wonder if it is possible to edit a QGEP in the field? That means:

- No internet connection (local copie)
- Tablet or even cellphone
- Editing 
- Synchronisation once your bake from the field

I tried QField, but I had a bunch of error messages and all the relations are destroyed, since it copies the layers to a gpkg.

Anyone has tried this? 
Thank you for your inputs.

**Comment by TriformMoritz** [comment]
Found this issue, form 3 years ago;
https://github.com/QGEP/QGEP/issues/412

**Reply by sjib** [reply]
@TriformMoritz: Contact @varrieta - I think she has the most experience with using qfield and qgep

**Comment by sjib** [comment, marked as answer]
Here I think you can find better documentation: https://github.com/QGEP/docs/issues/212
